library/ceph_volume.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
```

Comment:
no tests for that module?

Reply:
You're right, I probably should add tests. The logic wasn't terribly complicated so I didn't bother though.

Reply:
I've added some tests now.